maxim/logger/components/generation.py

```
@@ -548,10 +548,14 @@ def convert_chat_completion(chat_completion: Dict[str, Any]):
                 {
                     "index": choice.get("index", 0),
                     "message": {
-                        "role": choice.get("message").get("role", "assistant"),
-                        "content": choice.get("message").get("content", ""),
-                        "tool_calls": choice.get("message").get("tool_calls", None),
-                        "function_calls": choice.get("message").get(
+                        "role": (choice.get("message", {}) or {}).get(
+                            "role", "assistant"
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)